docs/reference/module-options.md

```diff
@@ -811,3 +843,198 @@ module.exports = {
 }
 ```
 
+
+## Options for widget modules
+
+Option settings in this section apply to all widgets (modules that extend `@apostrophecms/widget-type`).
+
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`className`](#classname) | String | A project-level class to apply to core widget templates. |
```

Comment:
I don't think we need to say 'project-level'. "Applies a class to core widget templates"